### PR TITLE
[sortinghat] Add poetry support

### LINKED ISSUE #462: Poetry support
We're replacing the way packages are generated in GrimoireLab. The new method will be using poetry. The goal of this task is to add support to it.

### PR DESCRIPTION
**Pull request by vchrombie**
This commit adds the files required to add poetry support to sortinghat muggle branch.

This update is required to start using the Bitergia/release-tools for automating the GrimoireLab toolset releases.

Fixes #462

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+requests = "^2.7.0"
+uri = "^2.0.1"
+jinja2 = "2.11.1"
+rq = "^1.4.0"
+django-rq = "^2.3.2"
+pandas = "^1.0.5"
+grimoirelab_toolkit = "^0.1.8"
+
+[tool.poetry.dev-dependencies]
+httpretty = "0.9.7"
```
Comment (sduenas) [inline pyproject.toml:56]: `httpretty` isn't needed for testing sortinghat.

**Review by sduenas — CHANGES_REQUESTED** [review]
Please check my comment.

File: pyproject.toml (hunk 1 above)
Comment (vchrombie) [reply pyproject.toml]: Ok

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**